Sources/FirebaseAuth/FirebaseAuthError.swift

```diff
@@ -1,117 +1,151 @@
+  public static let secondFactorAlreadyEnrolled: Self = .init(firebase.auth.kAuthErrorSecondFactorAlreadyEnrolled)
+  public static let maximumSecondFactorCountExceeded: Self = .init(firebase.auth.kAuthErrorMaximumSecondFactorCountExceeded)
+  public static let unsupportedFirstFactor: Self = .init(firebase.auth.kAuthErrorUnsupportedFirstFactor)
+  public static let emailChangeNeedsVerification: Self = .init(firebase.auth.kAuthErrorEmailChangeNeedsVerification)
   #if INTERNAL_EXPERIMENTAL
-  case invalidEventHandler = 79
-  case federatedProviderAlreadyInUse = 80
-  case invalidAuthenticatedUserData = 81
-  case federatedSignInUserInteractionFailure = 82
-  case missingOrInvalidNonce = 83
-  case userCancelled = 84
-  case unsupportedPassthroughOperation = 85
-  case tokenRefreshUnavailable = 86
+  public static let invalidEventHandler: Self = .init(firebase.auth.kAuthErrorInvalidEventHandler)
+  public static let federatedProviderAlreadyInUse: Self = .init(firebase.auth.kAuthErrorFederatedProviderAlreadyInUse)
+  public static let invalidAuthenticatedUserData: Self = .init(firebase.auth.kAuthErrorInvalidAuthenticatedUserData)
+  public static let federatedSignInUserInteractionFailure: Self = .init(firebase.auth.kAuthErrorFederatedSignInUserInteractionFailure)
+  public static let missingOrInvalidNonce: Self = .init(firebase.auth.kAuthErrorMissingOrInvalidNonce)
+  public static let userCancelled: Self = .init(firebase.auth.kAuthErrorUserCancelled)
+  public static let unsupportedPassthroughOperation: Self = .init(firebase.auth.kAuthErrorUnsupportedPassthroughOperation)
+  public static let tokenRefreshUnavailable: Self = .init(firebase.auth.kAuthErrorTokenRefreshUnavailable)
   #endif
 
   // Errors that are not represented in the C++ SDK, but are
   // present in the reference API.
-  case missingAndroidPackageName = 17037
-  case webNetworkRequestFailed = 17061
-  case webSignInUserInteractionFailure = 17063
-  case localPlayerNotAuthenticated = 17066
-  case nullUser = 17067
-  case gameKitNotLinked = 17076
-  case secondFactorRequired = 17078
-  case blockingCloudFunctionError = 17105
-  case internalError = 17999
-  case malformedJWT = 18000
+  public static let missingAndroidPackageName: Self = .init(rawValue: 17037)
+  public static let webNetworkRequestFailed: Self = .init(rawValue: 17061)
+  public static let webSignInUserInteractionFailure: Self = .init(rawValue: 17063)
+  public static let localPlayerNotAuthenticated: Self = .init(rawValue: 17066)
+  public static let nullUser: Self = .init(rawValue: 17067)
+  public static let gameKitNotLinked: Self = .init(rawValue: 17076)
+  public static let secondFactorRequired: Self = .init(rawValue: 17078)
+  public static let blockingCloudFunctionError: Self = .init(rawValue: 17105)
+  public static let internalError: Self = .init(rawValue: 17999)
+  public static let malformedJWT: Self = .init(rawValue: 18000)
```

Comment:
Please convert these to `public static var` where they are computed properties.

Reply:
Are you trying to avoid taking up space for the stored properties? I agree that computed properties would be better for that reason. Is that your thinking??